cfg/config.go

```diff
@@ -0,0 +1,192 @@
+/*
+ * knoxite
+ *     Copyright (c) 2020, Nicolas Martin <penguwin@penguwin.eu>
+ *
+ *   For license see LICENSE
+ */
+package cfg
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	gap "github.com/muesli/go-app-paths"
+)
+
+const appName = "knoxite"
+
+// Available configuration backends
+const (
+	CryptoConf = iota
```

Comment:
I think we should keep the default config option (FileConf) the first in order here.

Reply:
Reordered the configuration backends to make `FileConf` the first item in order